api/app/src/Repository/ClientRepository.php

```diff
@@ -135,20 +135,20 @@ public function findByCaseNumber(string $caseNumber): ?Client
             ->getOneOrNullResult();
     }
 
-    public function findByCaseNumberIncludingDischarged(string $caseNumber): ?Client
+    public function findByCaseNumberIncludingDischarged(string $caseNumber): mixed
     {
         $filter = $this->_em->getFilters()->getFilter('softdeleteable');
         $filter->disableForEntity(Client::class);
 
-        $client = $this
+        $clients = $this
```

Comment:
shall we check if results > 1 or does getResult do that (eg exacts one result)?

Reply:
getResult() returns a resultset, which might have multiple rows in it.